README.md

```diff
@@ -3,15 +3,15 @@
 
 A GitHub action to read and get data from the `CHANGELOG.md` file :rocket:
 
-**This action only work if your `CHANGELOG.md` file follows the [_Keep a Changelog_](https://github.com/olivierlacan/keep-a-changelog) standard for now.**
+**This action only works if your `CHANGELOG.md` file follows the [_Keep a Changelog_](https://github.com/olivierlacan/keep-a-changelog) standard for now.**
 
 ## Usage
 ### Pre-requisites
 Create a workflow `.yml` file in your repositories `.github/workflows` directory. An [example workflow](#example-workflow---upload-a-release-asset) is available below. For more information, reference the GitHub Help Documentation for [Creating a workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file).
 
 ### Inputs
 
-- `path`: The path the action can find the CHANGELOG. Optional. Defaults to `./CHANGELOG.md`.
+- `path`: The path the action can find the [CHANGELOG](CHANGELOG.md). Optional. Defaults to `./CHANGELOG.md`.
```

Comment:
I don't think we should put a link here because it is not our CHANGELOG that talk about but theirs. Also the default value already show where to find the file.

Reply:
Oops my bad 😅